### ISSUE TITLE
Configuration for rule "import/no-cycle" is invalid

### ISSUE THREAD

**Issue by btd1337** [iamturns/eslint-config-airbnb-typescript#111]
Using the new [Eslint 7.3.0](https://github.com/eslint/eslint/releases/tag/v7.3.0) version, this error is reported:

```log
.eslintrc.js »
 eslint-config-airbnb-typescript/base »
node_modules/eslint-config-airbnb-base/index.js » 
/opt/desenvolvimento/haizen/bridi-back/node_modules/eslint-config-airbnb-base/rules/imports.js: 
Configuration for rule "import/no-cycle" is invalid: 
Value null should be integer. 
```

**Comment by koba-yoshi** [comment]
https://github.com/iamturns/eslint-config-airbnb-typescript/issues/111 の問題が発生するのでマージは保留

**Comment by AdrienLemaire** [comment]
Related to https://github.com/airbnb/javascript/issues/2245

**Comment by iamturns** [comment]
v9.0.0 has just been released, which includes support for ESLint v7. Cheers!
